frontend/src/setupTests.ts

```diff
@@ -29,3 +29,14 @@ var localStorageMock = (function() {
 Object.defineProperty(window, 'localStorage', {
   value: localStorageMock,
 });
+
+//workaround to allow polyline and other svg map renderers to function correctly in tests.
+var createElementNSOrig = (global as any).document.createElementNS;
```

Comment:
what was occurring before?

Reply:
there were no tests that triggered polylines before.